tests/help.sh

```diff
@@ -3,17 +3,44 @@
 
 set -euo pipefail
 
+hidden=
 fail=0
 total=0
 checklen() {
-    local length subcommand subcommands
+    local length subcommand subcommands docfile doc_section longopts shortopts opt
     total=$((${total} + 1))
     echo "Checking coreos-installer $*..."
+
     length=$(target/debug/coreos-installer $* --help | wc -L)
     if [ "${length}" -gt 80 ] ; then
         echo "$* --help line length ${length} > 80"
         fail=1
     fi
+
+    longopts=$(target/debug/coreos-installer $* --help | \
+        (grep -Eo -- "--[a-zA-Z0-9_-]+" ||:) | \
+        (grep -Ev -- "--(help|version)" ||:))
+    shortopts=$(target/debug/coreos-installer $* --help | \
+        (grep -Eo -- "-[a-zA-Z0-9]," ||:) | \
+        tr -d , | \
+        (grep -Ev -- "-(h|V)" ||:))
+    # Ignore hidden subcommands, subcommands that are only containers for
+    # other subcommands, and subcommands with no options
```

Comment:
They should all have at least `-h`/`--help`, right? We can use that to verify that our grep was actually successful and we didn't regress on a format change.